src/ApiService/ApiService/onefuzzlib/ReproOperations.cs

```diff
@@ -1,16 +1,45 @@
-﻿using ApiService.OneFuzzLib.Orm;
+﻿using System.Threading.Tasks;
+using ApiService.OneFuzzLib.Orm;
 
 namespace Microsoft.OneFuzz.Service;
 
-public interface IReproOperations : IStatefulOrm<Repro, VmState> {
+public interface IReproOperations : IStatefulOperations<Repro, VmState> {
     public IAsyncEnumerable<Repro> SearchExpired();
 
     public System.Threading.Tasks.Task Stopping(Repro repro);
 
     public IAsyncEnumerable<Repro> SearchStates(IEnumerable<VmState>? States);
+
+    public IAsyncEnumerable<Repro> SearchByTimeRange((DateTimeOffset min, DateTimeOffset max) timeRange);
+
+    public Async.Task<ResultVoid<(int, string)>> Replace(Repro updatedRepro);
+}
+
+public interface IReproOrm : IOrm<Repro> {
```

Comment:
I don't understand the need for this indirection. The IOrm and IStatefulOrm are meant to abstract the operations on the each specific entities. The `IStatefulOrm` is just only adds state management to the existing operations.